ow_lander/scripts/arm_action_servers.py

```diff
@@ -0,0 +1,666 @@
+#!/usr/bin/env python2
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import rospy
+import actionlib
+from ow_lander.msg import *
+from LanderInterface import MoveItInterface
+from LanderInterface import LinkStateSubscriber
+from trajectory_async_execution import TrajectoryAsyncExecuter
+import all_action_trajectories
+from moveit_msgs.msg import RobotTrajectory
+from controller_manager_msgs.srv import SwitchController
+from ground_detection import GroundDetector
+from actionlib_msgs.msg import GoalStatus
+from geometry_msgs.msg import Point
+
+
+
+class UnstowActionServer(object):
+    
+    def __init__(self,name):
+        self._action_name = name
+        self._server = actionlib.SimpleActionServer(self._action_name, 
+                                                    ow_lander.msg.UnstowAction, 
+                                                    execute_cb=self.on_unstow_action, 
+                                                    auto_start = False)
+        self._server.start()
+        # Action Feedback/Result
+        self._fdbk = ow_lander.msg.UnstowFeedback()
+        self._result = ow_lander.msg.UnstowResult()
+        self._current_link_state = LinkStateSubscriber()
+        self._interface = MoveItInterface()
+        self._timeout = 0.0
+        self.trajectory_async_executer = TrajectoryAsyncExecuter()
+        self.trajectory_async_executer.connect("arm_controller")
+        
+    
+    def _update_feedback(self):
+ 
+        self._ls =  self._current_link_state._link_value
+        self._fdbk.current.x = self._ls.x
+        self._fdbk.current.y = self._ls.y
+        self._fdbk.current.z = self._ls.z
+        self._server.publish_feedback(self._fdbk)
+
+        
+    def _update_motion(self):
+
+        print("Unstow arm activity started")
```

Comment:
Would rather see rospy logging used for all print functions.
```suggestion
        rospy.loginfo("Unstow arm activity started")
```